src/components/dialogs/line-creation-dialog.js

```diff
@@ -4,27 +4,33 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-import { TextField } from '@material-ui/core';
 import Button from '@material-ui/core/Button';
 import Dialog from '@material-ui/core/Dialog';
 import DialogActions from '@material-ui/core/DialogActions';
 import DialogContent from '@material-ui/core/DialogContent';
 import DialogTitle from '@material-ui/core/DialogTitle';
 import Grid from '@material-ui/core/Grid';
-import { makeStyles } from '@material-ui/core/styles';
 import { useSnackbar } from 'notistack';
 import PropTypes from 'prop-types';
-import React, { useState } from 'react';
-import { FormattedMessage, useIntl } from 'react-intl';
+import React, { useCallback } from 'react';
+import { FormattedMessage } from 'react-intl';
 import { useParams } from 'react-router-dom';
 import {
     displayErrorMessageWithSnackbar,
     useIntlRef,
 } from '../../utils/messages';
 import { createLine } from '../../utils/rest-api';
-import TextFieldWithAdornment from '../util/text-field-with-adornment';
-import { validateField } from '../util/validation-functions';
-import ConnectivityEdition from './connectivity-edition';
+import {
+    filledTextField,
+    SusceptanceAdornment,
+    OhmAdornment,
+    AmpereAdornment,
+    useDoubleValue,
+    useInputForm,
+    useTextValue,
+    useConnectivityValue,
+} from './input-hooks';
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
This line is just a moved from over `import { useSnackbar } from 'notistack';` . Why ?

Reply:
Fixed